config/katello.migrations/171030135054-foreman-proxy-update.rb

```diff
@@ -0,0 +1,5 @@
+# foreman_proxy defaults for templates
+
+answers['foreman_proxy']['http'] = true if answers['foreman_proxy'].key?('http')
+answers['foreman_proxy']['http_port'] = '8000' unless answers['foreman_proxy'].key?('http_port')
```

Comment:
The default is an integer ```8000```. This converts it to a string but doesn't really add anything so can be removed.